runtime/common/CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ set_source_files_properties(
     JIT.cpp
     Logger.cpp
     RuntimeMLIR.cpp
-  PROPERTIES COMPILE_FLAGS -fno-rtti
+#  PROPERTIES COMPILE_FLAGS -fno-rtti
```

Comment:
If the compiler is happy with this, I guess we can just delete this line (and the comment above) :)

Reply:
thanks, will do!

Reply:
done.